src/utils/rasterOptionFunctions.js

```diff
@@ -124,12 +125,14 @@ export function getColorMaxFromStyle(style) {
 export function composeStyleString(color, min, max) {
   let str = "";
   str += color;
-  if (min && min !== "") {
-    str += ":" + min;
-
-    // we can only add max if min is also added
-    if (max && max !== "") {
-      str += ":" + max;
+  if (min || max) {
+    str += ":"
+    if (min) {
+      str += min
+    }
+    str += ":"
+    if (max) {
+      str += max
```

Comment:
Gaat fout als min of max 0 zijn, wat een valide waarde is. Of zijn het strings? Je hebt geen Typescript gebruikt.

Reply:
I think it functionaly works well but agree that we could benefit from typescript here.
Anyway since this is a isolated component I would like to do this in a next ticket.
We are probably best of by merging this asap.

Reply:
commented in de code.
Created JIRA ticket:
https://nelen-schuurmans.atlassian.net/browse/FRNT-320